### PR TITLE
:book: exe suffix to windows URL of clusterawsadm utility

### LINKED ISSUE #9795: Fix download instructions for clusterawsadm in the quick start
### What steps did you take and what happened?

`curl.exe -L https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v2.3.0/clusterawsadm-windows-amd64 -o clusterawsadm.exe`

I tried above command in windows to install clusterawsadm & clusterctrl cli's which failed in proper software installation.

`PS C:\Program Files (x86)> clusterawsadm
Program 'clusterawsadm.exe' failed to run: The specified executable is not a valid application for this OS platform.At line:1 char:1`

### What did you expect to happen?

`PS C:\Program Files (x86)> clusterawsadm
clusterawsadm provides helpers for bootstrapping Kubernetes Cluster`

I tweaked the installation command to `curl.exe -L https://github.com/kubernetes-sigs/cluster-api-provider-aws/releases/download/v2.3.0/clusterawsadm-windows-amd64.exe -o clusterawsadm.exe` and it worked

### Cluster API version

v1beta1

### Kubernetes version

1.21

### Anything else you would like to add?

_No response_

### Label(s) to be applied

/kind bug
One or more /area label. See https://github.com/kubernetes-sigs/cluster-api/labels?q=area for the list of labels.

### PR DESCRIPTION
**Pull request by vigneshp826**
**What this PR does / why we need it**:
Document correction to avoid clusterawsadm cli installation issue for windows environment.

**Which issue(s) this PR fixes** 
Fixes #9795

/area documentation

### CODE REVIEW

File: docs/book/src/user/quick-start.md [hunk 1]
```diff
@@ -417,7 +417,7 @@ clusterctl init --infrastructure aws
 
 Download the latest release; on Windows, type:
 ```
-curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64" version:">=2.0.0"}} -o clusterawsadm.exe
+curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64.exe" version:">=2.0.0"}} -o clusterawsadm.exe
```
Comment (chrischdi) [inline docs/book/src/user/quick-start.md:420]:
```suggestion
curl.exe -L {{#releaselink repo:"https://github.com/kubernetes-sigs/cluster-api-provider-aws" gomodule:"sigs.k8s.io/cluster-api-provider-aws" asset:"clusterawsadm-windows-amd64.exe" version:">=2.0.0"}} -o clusterawsadm.exe
```

I think this needs to be the capa repo, also for the other OS's for clusterawsadm.

The link generated is https://github.com/kubernetes-sigs/cluster-api/releases/download/v2.3.0/clusterawsadm-windows-amd64.exe and still does not work for me.

Comment (sbueringer) [reply docs/book/src/user/quick-start.md:420]: Yes!

Comment (vigneshp826) [reply docs/book/src/user/quick-start.md:420]: Sorry guys. Completely missed this PR. Any task pending from my side?

Comment (sbueringer) [reply docs/book/src/user/quick-start.md:420]: Yes https://github.com/kubernetes-sigs/cluster-api/pull/9870/files#r1424943989

The binary is hosted in the release artifacts of the cluster-api-provider-was repo so we can't use cluster-api as the repo

**Comment by chrischdi** [comment]
/close

in favor of: 

- #10170

**Pull request closed without merging**